1903_Samples/CppWinRT_Win32_App/SampleLibraryCS/Package.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+using Microsoft.ReactNative;
+using Microsoft.ReactNative.CSharp;
+
```

Comment:
Why do you need this Package.cs? Why do we need ReactNative?

Reply:
Upps,